Triangulation_3/include/CGAL/Triangulation_hierarchy_3.h

```diff
@@ -91,6 +85,11 @@ class Triangulation_hierarchy_3
   using Tr_Base::geom_traits;
 
 private:
+  // parameterization of the hierarchy
+  // maximal number of points is 30^5 = 24 millions !
+  static const int ratio = 30;
+  static const size_type minsize = 20;
+  static const int maxlevel = 5;
```

Comment:
In C++14, you are supposed to add a definition at namespace scope (outside the class). Otherwise you may get undefined symbol errors (usually in debug mode). In C++17, you can use `constexpr` to avoid that. Before C++17, an enum was a convenient way to avoid the extra definition.